libraries/exoplayer/src/test/java/androidx/media3/exoplayer/analytics/DefaultAnalyticsCollectorTest.java

```diff
@@ -2199,6 +2200,15 @@ public void onLoadStarted(
       reportedEvents.add(new ReportedEvent(EVENT_LOAD_STARTED, eventTime));
```

Comment:
you might need to use a different event type here, so we can test that both the new method **and** the old method are called

Reply:
Any naming convention to suggest here? I was thinking `EVENT_ON_LOAD_STARTED` but not sure if that is confusing and outside of Google's naming conventions

Reply:
What about keeping `EVENT_LOAD_STARTED` for the new un-deprecated one and renaming the old one to `EVENT_LOAD_STARTED_WITHOUT_RETRY_COUNT`? It's a bit long, but it only needs to live as long as we keep the deprecated method.

Reply:
Ah because i was reading this in a test, i thought this constant was defined in the test - my bad, now i see it's defined in `AnalyticsListener` interface.

In that case i retract my comment, we shouldn't duplicate this constant imo - we should find a different way to test that both the new and old methods are called.